02-peer-protocol.md

```diff
@@ -50,10 +67,374 @@ pubkey corresponding to the funding output nothing prevents duplicative channel
 ids.
 
 
+### `channel_id`, v2
+
+For channels established using the v2 protocol, the `channel_id` is the
+SHA256(lesser-revocation-basepoint || greater-revocation-basepoint),
+where the lesser and greater is based off the order of the basepoint.
```

Comment:
Ah yeah, good point (hehe). Since the definition in `01` is for the word "point", it seems that would apply to base*points* as well. I'll remove the reference to the encoding type here.